sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/metadata.yaml

```
@@ -0,0 +1,23 @@
+---
+friendly_name: Data from external source Adjust
+description: >
+  This table represents data downloaded from Adjust. Please note that the API token used is Marlene Hirose's personal Adjust API token
+owners:
+  - mhirose@mozilla.com
+labels:
+  incremental: false
+scheduling:
+  dag_name: bqetl_adjust_derived
+  date_partition_parameter: submission_date
+  parameters:
```

CI is failing because there are some parameters missing here

---

Are the parameters needed?

---

not in this case, instead you need to specify script arguments, like this: https://github.com/mozilla/bigquery-etl/blob/dc482ad8d5ee6de28c0d5a997e83d2a0a563981f/sql/moz-fx-data-marketing-prod/iprospect/adspend_v1/metadata.yaml#L15

```suggestion
  arguments: ["--date", "{{ ds }}"]
```